src/main/java/mil/dds/anet/database/MartImportedReportDao.java

```diff
@@ -0,0 +1,116 @@
+package mil.dds.anet.database;
+
+import java.util.List;
+import mil.dds.anet.beans.lists.AnetBeanList;
+import mil.dds.anet.beans.mart.MartImportedReport;
+import mil.dds.anet.database.mappers.MartImportedReportMapper;
+import mil.dds.anet.utils.DaoUtils;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.statement.Query;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Component
+public class MartImportedReportDao {
+
+  protected final DatabaseHandler databaseHandler;
+
+  public MartImportedReportDao(DatabaseHandler databaseHandler) {
+    this.databaseHandler = databaseHandler;
+  }
+
+  protected Handle getDbHandle() {
+    return databaseHandler.getHandle();
+  }
+
+  protected void closeDbHandle(Handle handle) {
+    databaseHandler.closeHandle(handle);
+  }
+
+  @Transactional
+  public List<MartImportedReport> getAll() {
+    return getAll(0, 0).getList();
+  }
+
+  @Transactional
+  public MartImportedReport getByReportUuid(String reportUuid) {
+    final Handle handle = getDbHandle();
+    try {
+
+      return handle
+          .createQuery(
+              "/* MartImportedReportGetByReportUuid*/ SELECT * FROM \"martImportedReports\" "
+                  + "WHERE \"reportUuid\" = :reportUuid")
+          .bind("reportUuid", reportUuid).map(new MartImportedReportMapper()).findFirst()
+          .orElse(null);
+    } finally {
+      closeDbHandle(handle);
+    }
+  }
+
+  @Transactional
+  public AnetBeanList<MartImportedReport> getAll(int pageNum, int pageSize) {
+    final Handle handle = getDbHandle();
+    try {
+      final StringBuilder sql = new StringBuilder(
+          "/* MartImportedReportCheck */ SELECT * FROM \"martImportedReports\" ORDER BY \"receivedAt\" ASC");
+      sql.insert(0, "SELECT *, COUNT(*) OVER() AS \"totalCount\" FROM (");
+      sql.append(") AS results");
+      sql.append(" ORDER BY sequence DESC");
+      if (pageSize > 0) {
+        sql.append(" OFFSET :offset LIMIT :limit");
+      }
+      final Query query = handle.createQuery(sql);
+      if (pageSize > 0) {
+        query.bind("offset", pageSize * pageNum).bind("limit", pageSize);
+      }
+      return new AnetBeanList<>(query, pageNum, pageSize, new MartImportedReportMapper());
+    } finally {
+      closeDbHandle(handle);
+    }
+  }
+
+  @Transactional
+  public AnetBeanList<MartImportedReport> getAllSequences(List<Long> sequences) {
+    final Handle handle = getDbHandle();
+    try {
+      final Query query = handle.createQuery(
+          "/* MartImportedReportSequencesCheck */ SELECT * FROM \"martImportedReports\" WHERE \"sequence\" IN (<sequences>)")
+          .bindList("sequences", sequences);
```

Comment:
You should always use:
```java
          .bindList(NULL_KEYWORD, "sequences", sequences);
```
to be safe (else `sequences` is not allowed to be empty).
Import:
```java
import static org.jdbi.v3.core.statement.EmptyHandling.NULL_KEYWORD;
```